mne_bids_pipeline/scripts/preprocessing/_06_ptp_reject.py

```diff
@@ -62,10 +63,27 @@ def drop_ptp(*, cfg, subject, session, in_files):
 
     # Get rejection parameters and drop bad epochs
     epochs = mne.read_epochs(in_files.pop('epochs'), preload=True)
-    reject = config.get_reject(epochs=epochs)
+    reject = _get_reject(
+        subject=subject,
+        session=session,
+        reject=cfg.reject,
+        ch_types=cfg.ch_types,
+        param='reject',
+        epochs=epochs,
```

Comment:
... and another bugfix (I think) -- these epochs are already decimated, but when `autoreject_global` was used, `decim=decim` was being passed so they were being decimated again.